upstream-community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,454 @@
+          },
+          "spec": {
+            "requirements": false,
+            "serviceAccount": "aqua-sa",
+            "deploy": {
+              "replicas": 1,
+              "service": "ClusterIP",
+              "image": {
+                "repository": "scanner",
+                "registry": "registry.aquasec.com",
+                "tag": "4.0",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "login": {
+              "username": "administrator",
+              "password": "Password1",
+              "host": "http://aqua-server-svc:8080"
+            },
+            "openshift": false
+          }
+        },
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaServer",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "add": false,
+            "serviceAccount": "aqua-sa",
+            "dbSecretName": "aqua-database-password",
+            "dbSecretKey": "db-password",
+            "rbac": {
+              "enable": true,
+              "openshift": false,
+              "privileged": true
+            },
+            "deploy": {
+              "replicas": 1,
+              "service": "LoadBalancer",
+              "image": {
+                "repository": "console",
+                "registry": "registry.aquasec.com",
+                "tag": "3.5",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "aquaDb": "aqua-database",
+            "adminPassword": "Password1",
+            "licenseToken": null,
+            "clusterMode": true,
+            "ssl": false,
+            "auditSsl": false,
+            "dockerless": false,
+            "openshift": false
+          }
+        }
+      ]
+spec:
+  displayName: Aqua Security Operator
+  version: 0.0.1
+  description: |-
+    The Aqua Security Operator runs within a Kubernetes or Openshift cluster and provides a means to deploy and manage Aqua Security cluster and components as
+    * Server (Console)
+    * Gateway
+    * Database (Not recommended for production environments)
+    * Enforcer (Agent)
+    * Scanner CLI
+    * CSP (Package of Server, Gateway and Database)
+
+    Use the aqua-operator to - 
+    * Deploy Aqua Security components on Kubernetes or Openshift
+    * Scale up Aqua Security components with extra replicas
+    * Assign metadata tags to Aqua Security components
+    * Automatic scale to the Aqua Scanner CLI by the count of the Scanning Queue.
+
+    ## Before You Begin
+    ### Openshift 3.11+
+    you need to create - 
+      * Service account with pull permissions from aqua docker registry
+      * Secret for docker registry 
+      * Secret for database
+
+    ```
+    oc new-project aqua
+
+    oc create serviceaccount aqua-sa -n aqua
+
+    oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:aqua:aqua-sa -n aqua
```

Comment:
For the OpenShift (community-operators) version, this can be added into the CSV so that OLM can create this service account and roles automatically

Reply:
How can I add it?!

Reply:
I'm referring to the service account and cluster-roles and the SecurityContextConstraints. The secrets can stay though.
Also, this description is for OpenShift but is in the wrong CSV.
`upstream-community-operators` is for vanilla Kubernetes
`community-operators` is for OpenShift

Reply:
ok

Reply:
Description **Updated!**

but about the SecurityContextConstraints how can i create it in the *Operator Install* Section.
i don't find in the olm docs a way to create it. 

and about this requirments. you dont need it to install the operator its only for using the CRDs then you needto create this before running aqua.

its not for the operator!

Reply:
So its only after the operator its installed